### PR TITLE
Rebased deleteforwardurl

### PR DESCRIPTION
**Pull request by imqt**
This PR contains changes for deleting forward url suggestions and small fixes for hotkeys functionalities (shift and ctrl)

### CODE REVIEW

File: src/routes/Viewport/Workspace/Dashboard/Body/Sidebar/Webhooks/index.tsx [hunk 1]
```diff
@@ -103,6 +105,8 @@ const Webhooks = ({ filter }: { filter: FilterEnum }) => {
   const isItemLoaded = (index: number) =>
     !hasNextPage || index < filteredWebhooks.length;
 
+  console.log('Number of webhook ids: ' + selectedWebhookIds.length);
```
Comment (johnnyoshika) [inline src/routes/Viewport/Workspace/Dashboard/Body/Sidebar/Webhooks/index.tsx]: `console.log` should be removed.

File: src/hooks/types/DeleteForwardUrls.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  affectedRows: number;
+}
+
+export interface DeleteForwardUrls {
+  deleteForwardUrls: DeleteForwardUrls_deleteForwardUrls;
+}
+
+export interface DeleteForwardUrlsVariables {
+  input: DeleteForwardUrlInput;
+}
```
Comment (johnnyoshika) [inline src/hooks/types/DeleteForwardUrls.ts]: This file is not needed.

File: src/hooks/useFetchForwardUrls.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+`;
+
+export const GET_FORWARD_URLS = gql`
+  query GetForwardUrls($endpointId: ID!) {
+    forwardUrls(endpointId: $endpointId) {
+      ...ForwardUrl
+    }
+  }
+  ${FORWARD_URL_FRAGMENT}
+`;
```
Comment (johnnyoshika) [inline src/hooks/useFetchForwardUrls.tsx]: * All queries have been moved to `src/schema/queries.ts` and fragments to `src/schema/fragments.ts`, so these pieces should move there as well
* As an FYI, there's nothing `hook` like in this file, so the filename shouldn't start with `use`

File: src/components/Autosuggest/index.tsx [hunk 4]
```diff
@@ -162,6 +194,7 @@ const Autosuggest = ({
       <SuggestionsList
         activeIndex={state.activeIndex}
         filteredSuggestions={state.filteredSuggestions}
+        deleteForwardUrls={deleteForwardUrls}
         onMouseDown={onMouseDown}
       />
     </div>
```
Comment (johnnyoshika) [inline src/components/Autosuggest/index.tsx:200]: There's a strange bug. Any time I do something with a forward URL, the selected endpoint in the AutoForward dropdown resets to the first item in the list. This happens when I:
* Forward a webhook
* Delete a forward URL from the main webhook panel
* Delete a forward URL from the auto-forward panel

Here's a recording that demonstrates this: https://recordit.co/9QNBWX9GDv

**Review by johnnyoshika — CHANGES_REQUESTED** [review]
Thanks guys! Looking really good. Here are a few issues that I noticed for your attention.

File: src/hooks/types/DeleteForwardUrls.ts (hunk 2 above)
Comment (awong503) [reply src/hooks/types/DeleteForwardUrls.ts]: Deleted file

File: src/hooks/useFetchForwardUrls.tsx (hunk 3 above)
Comment (awong503) [reply src/hooks/useFetchForwardUrls.tsx]: Looks like the queries and fragments are already there. deleted file.

File: src/components/Autosuggest/index.tsx (hunk 4 above)
Comment (imqt) [reply src/components/Autosuggest/index.tsx:200]: The issue is resolved. The selection doesn't get changed when we do the actions mentioned above anymore.

**Pull request merged**